### ISSUE TITLE
Email always added to default list as transactional

### ISSUE THREAD

**Issue by DeveloperWil** [restrictcontentpro/RCP-MailChimp#33]
tl;dr
Subscriber's email is being put onto two email lists; 1) the subscription level as "subscribed" 2) the default list as "transactional"

Issue:
In the RCP MailChimp settings the Default List says "Choose the list to subscribe users to if no per-level list is selected."

We have a different mailchimp list set for all of our subscription levels.  The email is correctly put on the per-level specified list, however, the email is also put onto the default list as transactional. 

It should only go on the list we specify on the subscription level or if none is specified use the system default in the RCP MailChimp Pro settings.  Not both.

**Comment by nosegraze** [comment]
Thanks @DeveloperWil !

I'm going to close this because this repo is for the free MailChimp add-on, which is different from the MailChimp Pro add-on. But I will copy this issue over into our MailChimp Pro repo.
